Two changes:

* Add an alias for `which', so abcde can find things in store
* Choose the right CDROM reader syntax for `cd-paranoia'

--- abcde-2.5.4/abcde~	2012-09-18 06:09:31.000000000 -0700
+++ abcde-2.5.4/abcde	2012-10-27 00:08:48.000862364 -0700
@@ -17,6 +17,11 @@
 
 VERSION='2.5.4'
 
+which ()
+{
+  type -P $1
+}
+
 usage ()
 {
 echo "This is abcde v$VERSION."
@@ -3497,6 +3502,10 @@
 	for DEFAULT_CDROMREADER in $DEFAULT_CDROMREADERS; do
 		if new_checkexec $DEFAULT_CDROMREADER; then
 			CDROMREADERSYNTAX=$DEFAULT_CDROMREADER
+                        case "$DEFAULT_CDROMREADER" in
+			  cd-paranoia)  CDROMREADERSYNTAX=cdparanoia;;
+			  *)            CDROMREADERSYNTAX=$DEFAULT_CDROMREADER;;
+			esac
 			break
 		fi
 	done
